src/components/Sections/Programs/tabs/Programs/List.js

```diff
@@ -35,13 +35,11 @@ const renderUnit = (unit, t) => {
 };
 
 const ProgramsList = () =>
-  <div>
-    <UnitList
-      key="programs-main"
-      namespace="programs-main"
-      renderUnit={renderUnit}
-      extraFetchParams={{ content_type: CT_VIDEO_PROGRAM_CHAPTER }}
-    />
-  </div>;
+  <UnitList
+    key="programs-main"
+    namespace="programs-main"
+    renderUnit={renderUnit}
+    extraFetchParams={{ content_type: CT_VIDEO_PROGRAM_CHAPTER }}
+  />;
```

Comment:
Are you sure the wrapping < div > was not required for CSS?

Reply:
redundant div